src/WebJobs/Config/ApplicationInsightsWebJobsBuilderExtensions.cs

```diff
@@ -60,6 +61,11 @@ public static IWebJobsBuilder AddApplicationInsights(this IWebJobsBuilder builde
                         MaxTelemetryItemsPerSecond = 20
                     };
 
+                    if (!config.GetSection(path).Exists())
```

Comment:
Add a comment here as to why we're doing this, maybe?

Reply:
Also -- go ahead bump this to the next preview version in this PR (unless you already have done that)